### ISSUE TITLE
Preserve significant white spaces in xml messages displayed in dashboard

### ISSUE THREAD

**Issue by AndreaPravato** [nextgenhealthcare/connect#4448]
**Is your feature request related to a problem? Please describe.**
Messages displayed in connect dashboard do not exactly correspond to actual data that was elaborated by the channel.
Significant white spaces (spaces within a text field), are not currently preserved in the xml representation displayed in dashboard. mystring=" " is displayed as `<mystring/>` in dashboard.

**Describe your use case**
I need to be able to copy the xml message from dashboard, related to a past elaboration, and replicate the exact same channel behavior on a channel clone, passing in input to that channel that xml message.

**Describe the solution you'd like**
Preserve significant white spaces in xml representation of messages that are displayed in dashboard.

**Additional context**
Screenshots related to the finding. A database reader reads a field (provincia_domicilio) that is made of a single white space " ". A source transformer converts to String. The channel goes on correctly using the white space, which is meaningful in the particular application. If I check the raw message in dashboard, though, I see `<provincia_domicilio/>`; I would expect to see  `<provincia_domicilio> <provincia_domicilio/>`
![SelectQuery](https://user-images.githubusercontent.com/4329975/104806216-3f352f00-57d6-11eb-8e6a-055141b21655.png)
![sourceTransformer](https://user-images.githubusercontent.com/4329975/104806219-44927980-57d6-11eb-9535-1535eb78d5f3.png)
![emptytag](https://user-images.githubusercontent.com/4329975/104806226-48be9700-57d6-11eb-89c8-546882547459.png)

**Comment by cturczynskyj** [comment]
Yeah, I agree, `<mystring> </mystring>` is not the same as `<mystring/>`. This is likely a bug in how the message is being "Pretty Printed". You can get around this by unchecking the "Format XML Messages" checkbox at the bottom of the window.
![image](https://user-images.githubusercontent.com/15792647/107833771-2d996580-6d49-11eb-80c2-b8dc8cbea71b.png)
